config/jobs/cert-manager/release-previous/cert-manager-release-previous-presubmits.yaml

```diff
@@ -537,8 +537,8 @@ presubmits:
   - name: pull-cert-manager-e2e-v1-18
     cluster: gke
     context: pull-cert-manager-e2e-v1-18
+    optional: false
```

Comment:
This is the 'previous' file - if we are switching the default Kubernetes e2e job from v1.17 to v1.18, we also need to update the branch protection config here: https://github.com/jetstack/testing/blob/a746b992b159beb6ddea486a3f383c060a4271d4/config/config.yaml#L36-L51

I think we'll want the same default across all branches come to think, unless we configure branch-specific merge protection config (which starts getting quite confusing..) 🙄